tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolLookupTest.java

```diff
@@ -393,6 +395,24 @@ public Object[][] getOnFailSymbolPositions() {
         };
     }
 
+    @Test
+    public void testSymbolLookupInModuleAlias() {
+        Project project = BCompileUtil.loadProject("test-src/symbol_lookup_with_module_alias_test.bal");
+        SemanticModel model = getDefaultModulesSemanticModel(project);
+        Document srcFile = getDocumentForSingleSource(project);
+        List<Symbol> visibleSymbols = model.visibleSymbols(srcFile, LinePosition.from(19, 25));
+        List<String> expectedModuleSymbols = List.of("obj", "tbl", "regexp");
```

Comment:
Why is array symbol not in the test?

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/38093/commits/e02ceb9e34746fd3d5f0f4518ce280c9cdedf891.